chapters/data-collection.tex

```diff
@@ -1,362 +1,251 @@
+but few covering the particularly challenges and opportunities presented by electronic surveys (often referred to as Computer Assisted Personal Interviews (CAPI)).
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Computer-Assisted\_Personal\_Interviews\_(CAPI)}}
+As there are many electronic survey tools, we focus on workflows and primary concepts, rather than software-specific tools.
+The chapter covers questionnaire design, piloting and programming; monitoring data quality during the survey; and how to ensure confidential data is handled securely from collection to storage and sharing.
+
+
 \end{fullwidth}
 
 %------------------------------------------------
 
 \section{Designing CAPI questionnaires}
+A well-designed questionnaire results from careful planning, consideration of analysis and indicators, close review of existing questionnaires, survey pilots, and research team and stakeholder review.
+Although most surveys are now collected electronically  --
+\textbf{Questionnaire design}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Questionnaire_Design}}
+\index{questionnaire design}
+(content development) and questionnaire programming (functionality development) should be seen as two strictly separate tasks.
+The research team should agree on all questionnaire content and design a paper version before programming a CAPI version.
+This facilitates a focus on content during the design process and ensures teams have a readable, printable paper version of their questionnaire.
+Most importantly, it means the research, not the technology, drives the questionnaire design.
+
+An easy-to-read paper version of the questionnaire is particularly critical for training enumerators, so they can get an overview of the survey content and structure before diving into the programming.
+It is much easier for enumerators to understand all possible response pathways from a paper version, than from swiping question by question.
+Finalizing the questionnaire before programming also avoids version control concerns that arise from concurrent work on paper and electronic survey instruments.
+Finally, a paper questionnaire is an important documentation for data publication.
+
+The workflow for designing a questionnaire will feel much like writing an essay, or writing pseudocode: begin from broad concepts and slowly flesh out the specifics. It is essential to start with a clear understanding of the
+\textbf{theory of change} \sidenote{\url{https://dimewiki.worldbank.org/wiki/Theory_of_Change}} and \textbf{experimental design} for your project.
+The first step of questionnaire design is to list key outcomes of interest, as well as the main covariates and variables needed for experimental design.
```

Comment:
We have not used the term _covariates_ earlier in the book. Use something else or explain it as it not an obvious term if you have not heard it before